### PR TITLE
How to shipping documentation

### PR DESCRIPTION
**Pull request by sebdak**

### CODE REVIEW

**Review by SvenErikPy — APPROVED** [review]
Beautiful

**Review by christiansyoung — APPROVED** [review]
This good!

**Comment by Ceredron** [comment]
Awesome! Should check out how it looks in Docusaurus post-merge too

**Review by Ceredron — APPROVED** [review]

File: vipps-checkout-how-it-works-shipping.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+![Shipping provicer default logo](resources/shipping_logo-default.png)
+
+## Enriching features
+For some of the providers we offer enriching features, including:
+- Pickup point
+- Home delivery
+
+These features change the user interface to allow for more specific selection of delivery time or place, where some of the features require you (the merchant) to provide credentials. Credentials are provided in the `logistics.integrations` property at session initiation.
+
+An enriching feature can be chosen by setting the `type` property on the logstics option, again refering to the [API spec](https://vippsas.github.io/vipps-developer-docs/api/checkout#tag/Session/paths/~1session/post) for which features are available for each shipping provider.
```
Comment (jorgensel) [inline vipps-checkout-how-it-works-shipping.md:31]:
```suggestion
An enriching feature can be chosen by setting the `type` property on the logistics option, again referring to the [API spec](https://vippsas.github.io/vipps-developer-docs/api/checkout#tag/Session/paths/~1session/post) for which features are available for each shipping provider.
```

File: vipps-checkout-how-it-works-shipping.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Pickup point
+
+The pickup point feature is enabled for by setting `type: "PICKUP_POINT"`. The title will become `{providerName} pick-up point` (e.g. "Posten pick-up point). The consumer gets to choose an available pickup point based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```
Comment (jorgensel) [inline vipps-checkout-how-it-works-shipping.md:35]:
```suggestion
The pickup point feature is enabled for by setting `type: "PICKUP_POINT"`. The title will become `{providerName} pick-up point` (e.g. "Posten pick-up point). The consumer gets to choose an available pickup point based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```
```suggestion
The pickup point feature is enabled for by setting `type: "PICKUP_POINT"`. The title will become `{providerName} pick-up point` (e.g. "Posten pick-up point). The consumer gets to choose an available pickup point based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```
```suggestion
The pickup point feature is enabled by setting `type: "PICKUP_POINT"`. The title will become `{providerName} pick-up point` (e.g. "Posten pick-up point). The consumer gets to choose an available pickup point based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```

File: vipps-checkout-how-it-works-shipping.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+![Pickup point animation](resources/shipping_pickup-point.gif)
+
+### Home delivery
+
+The home delivery feature is enabled by setting `type: "HOME_DELIVERY"`. The title will become `{providerName} home delivery` (e.g. "Porterbuddy home delivery") by default. The comsumer gets to choose an available delivery window based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```
Comment (jorgensel) [inline vipps-checkout-how-it-works-shipping.md:41]:
```suggestion
The home delivery feature is enabled by setting `type: "HOME_DELIVERY"`. The title will become `{providerName} home delivery` (e.g. "Porterbuddy home delivery") by default. The consumer gets to choose an available delivery window based on the address. Vipps relays the selected option as part of the content in the "session completed callback".
```

**Review by christiansyoung — APPROVED** [review]

**Comment by eirikv** [comment]
Should we write something about booking, that we do booking for porterbuddy and instabox but not for others?

**Review by eirikv — APPROVED** [review]

**Review by eirikv — APPROVED** [review]

**Review by eirikv — APPROVED** [review]

**Pull request merged**